extensions/kafka/src/test/java/com/hazelcast/jet/kafka/impl/KafkaTestSupport.java

```diff
@@ -113,6 +114,7 @@ private KafkaProducer<Integer, String> getProducer() {
             producerProps.setProperty("bootstrap.servers", brokerConnectionString);
             producerProps.setProperty("key.serializer", IntegerSerializer.class.getCanonicalName());
             producerProps.setProperty("value.serializer", StringSerializer.class.getCanonicalName());
+            producerProps.setProperty("max.block.ms", String.valueOf(KAFKA_MAX_BLOCK_MS));
```

Comment:
The default for this is 1 minute, this seems large enough to me, in what case is it not sufficient?

We have timeout 5 mins for each test, so 10 mins doesn't make sense (if the operation would take more than 5 min the test would timeout and fail anyway).

Reply:
There were cases that Jenkins overload made Kafka struggle to response in that time - the related issue is just Kafka not responding in time, probably due to Jenkins load as it's quite rare. This change was solely to minimise the risk of such Timeouts.

I see your point of value not making sense; I can switch it to 2 mins for example, but it's also rather arbitrary value I hope will help avoid the issue in the future. WDYT?

Reply:
I've reduced timeout to 2 mins